### ISSUE TITLE
No clear indicator in the header re: what page you're currently viewing

### ISSUE THREAD

**Issue by hilarysk** [nodunayo/speakerline#564]
Obviously the content changes for each page, but I think it would be helpful to highlight the currently active page in the navigation (especially if someone sends you a link to a specific page), e.g.:

 
<img width="1085" alt="Screen Shot 2024-05-08 at 11 18 46 AM" src="https://github.com/nodunayo/speakerline/assets/6457189/de87c985-9d93-440b-b556-18db3d8b28cc">
 
OR

<img width="1100" alt="Screen Shot 2024-05-08 at 11 17 49 AM" src="https://github.com/nodunayo/speakerline/assets/6457189/a8aea0c9-93ea-418a-b919-06f83173c713">

**Comment by nodunayo** [comment]
Yes, that, or maybe emboldened text/text of a different colour? Or your two suggestions above, I prefer the underline!

**Comment by hilarysk** [comment]
Working on this now :)
